0.data-download/data_downloader.ipynb

```diff
@@ -100,9 +100,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:gene_dependency_representations] *",
+   "display_name": "base",
    "language": "python",
-   "name": "conda-env-gene_dependency_representations-py"
+   "name": "python3"
```

Comment:
same goes for here too. We should be using the `gene_dependency_representations` kernel